pkg/api/http_server.go

```diff
@@ -42,24 +42,31 @@ func init() {
 	})
 }
 
+type ProvisioningService interface {
```

Comment:
I moved the interface here as you mentioned once @bergquist. I like the concept though not sure if there won't be lots of duplicate interfaces around.

Reply:
Do we have that somewhere already? Do you mean something like an `interfaces.go` file per package with all interfaces that are required by that package?